api/gateway.proto

```diff
@@ -275,4 +275,10 @@ message GatewayConnectionStats {
     uint32 count = 4;
   }
   RoundTripTimes round_trip_times = 9;
+
+  message SubBandStatistics {
```

Comment:
My suggestion would be:

```proto
message SubBandStats {
	uint64 min_frequency = 1;
	uint64 max_frequency = 2;
	float64 downlink_utilization_limit = 3;
	float64 downlink_utilization = 4;
}
```

I think it's more useful to give absolute utilization, so let's make the API return both the actual utilization and the maximum (the duty cycle limit).

We currently don't calculate uplink utilization of a sub-band, but we might want to do that in the future, so let's be prepared for that in our field naming.